docs/source/intro/troubleshooting.rst

```diff
@@ -77,6 +77,64 @@ A way to do it is to add a line similar to the following to the ``~/.bashrc`` an
 
 .. _Stackoverflow link: http://stackoverflow.com/questions/21079820/how-to-find-pg-config-pathlink
 
+RabbitMQ Installation
+.....................
+
+If in ``verdi status`` RabbitMQ is not connected, first check that RabbitMQ is actually running:
+
+.. code-block:: console
+
+   $ rabbitmqctl status
+   Status of node rabbit@ph-tsm15-025 ...
+   [{pid,86960},
+   ...
+   {listeners,[{clustering,25672,"::"},{amqp,5672,"::"},{http,15672,"::"}]},
+
+By default, AiiDA profiles are configured to connect to RabbitMQ *via* ``amqp://guest:guest@127.0.0.1:5672``, hence this port should be open for connections.
+In Linux / Mac OSX you can also check which ports a PID has open using:
+
+.. code-block:: console
+
+   $ lsof -Pan -p 86960 -i
```

Comment:
Same as above, `root` permission needed to access the port in rabbiltmq, if it is not installed user defined e.g. by conda-forge.